app/javascript/src/accessible-autocomplete.js

```diff
@@ -0,0 +1,16 @@
+import accessibleAutocomplete from 'accessible-autocomplete'
+
+accessibleAutocomplete.enhanceSelectElement({
+  defaultValue: '',
+  selectElement: document.querySelector('#non-uk-home-address-country-code-field')
```

Comment:
You may wish to use an attribute other than the `id` as this will change when there is an error on this input, thereby causing the component to fall back to using the `select` instead.

Reply:
Improvements: stop code running if select element is not found

Reply:
Good point on your first point

To your second point, i'd realised this was happening. I'm not sure what the best practice for doing this is. I'd had a quick google but hadn't got any conclusions. Do you know how best to this in a railsy way?